docs/guide/token_factory.md

```diff
@@ -0,0 +1,798 @@
+
+Congratulations, you have scaffolded an entire CRUD application. While the purpose of the token factory is to create denoms, you want to prevent an initialized denom from being deleted. You remove the delete function for denoms in the following steps.
+
+After scaffolding the denom map, it is a good time to make a first git commit:
+
+```bash
+git add .
+git commit -m "Add token factory module and denom map"
+```
+You can come back to this step in case something goes wrong with the following steps.
+
+## Remove Delete Messages Functionality
+
+Since a created denom is subsequently handled by the `bank` module like any other native denom, the denom must not be deletable. To prevent the denom from being deleted, remove all references to the delete action of the scaffolded CRUD type.
+
+In order to remove the functionality to delete token, you must remove these delete functions from proto, client, keeper, and handler as shown in the next sections:
+
+### Proto
+
+In the proto file `proto/tokenfactory/tx.proto`, remove this code:
+
+```proto
+rpc DeleteDenom(MsgDeleteDenom) returns (MsgDeleteDenomResponse);
+```
+
+Then, remove the `MsgDeleteDenom` and `MsgDeleteDenomResponse` messages.
+
+### Client
+
+Navigate to the client in `x/tokenfactory/client` and make these changes:
+
+- In the `x/tokenfactory/client/cli/tx_denom_test.go` file, remove the entire `TestDeleteDenom()` function.
+- In the `x/tokenfactory/client/cli/tx_denom.go` file, remove the entire `CmdDeleteDenom()` function.
+- In the `x/tokenfactory/client/cli/tx.go` file, remove the line that has the delete command:
+
+    ```go
+    cmd.AddCommand(CmdDeleteDenom())
+    ```
+
+### Keeper
+
+In the `keeper` directory, there are a few files contain the delete denom functionality:
+
+- In the `x/tokenfactory/keeper/denom_test.go` file, remove the `TestDenomRemove()` function.
+- In the `x/tokenfactory/keeper/denom.go` file, remove the entire `RemoveDenom()` function.
+- In the `x/tokenfactory/keeper/msg_server_denom_test.go` file, remove the `TestDenomMsgServerDelete()` function.
+- In the `x/tokenfactory/keeper/msg_server_denom.go` file, remove the `DeleteDenom()` function.
+
+### Types
+
+The `types` directory defines functions and validations that describe the format of the blockchain data. You must remove the delete denom functionality from the codec, the message denom test, and the message denom file.
+
+- Start with the codec in `x/tokenfactory/types/codec.go`, and remove the codec and interface registrations for `MsgDeleteDenom`.
+- In the `x/tokenfactory/types/messages_denom_test.go` test, remove the `TestMsgDeleteDenom_ValidateBasic()` function.
+- In the message denom `x/tokenfactory/types/messages_denom.go` file, remove the entire part that references `MsgDeleteDenom()`.
+
+### Handler
+
+In the handler, update the switch file for all of the messages.
+
+- Open the `x/tokenfactory/handler.go` file and remove `MsgDeleteDenom` case from `NewHandler` function.
```

Comment:
what do we mean by "case"?
do we mean "instance" or "code"?

Reply:
There is a `switch` statement there that has `case`s https://go.dev/tour/flowcontrol/9

Should be fine the way it is I think....